src/sections/share-data/view-request.jsx

```diff
@@ -36,20 +36,32 @@ const formatDate = (dateString) => {
 
 function mapShareableTypes(arrayOfObjects) {
   return arrayOfObjects?.map((obj) => {
-    if (obj.shareable_type === 'ContactInformation') {
-      return 'Contact';
-    } if (obj.shareable_type === 'EmploymentHistory') {
-      return 'Employment';
+    switch (obj.type) {
+      case 'basic_information':
+        return 'Basic Information';
+      case 'contact':
+        return 'Contact Information';
+      case 'education':
+        return 'Education';
+      case 'financial':
+        return 'Financial Information';
+      case 'identification':
+        return 'Identification';
+      case 'realestate':
+        return 'Real Estate';
+      case 'residential':
+        return 'Residential Information';
+      default:
+        return obj.type;
     }
-    return obj.shareable_type; 
   });
 }
 
 
 const ViewRequest = ({ data, handleCloseModal }) => {
  
   const [loading, setLoading] = useState()
-  const types = mapShareableTypes(data?.shareable_informations)

```

Comment:
why is this being changed

Reply:
shareable_informations is empty. the requested data is now in requested_info array